index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
- [ ] [OPTIONAL] Kindly, make sure you added some emojis 👍, to make it more friendly 😃.
- [ ] [OPTIONAL] Also, Kindly make sure to highlight the effort of the student 👍